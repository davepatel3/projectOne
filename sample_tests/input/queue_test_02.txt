float
2
2
2
ef -82.2401
ef 93.5007
ps
ef -1.15411
gc
ps
pf
gf
qt
