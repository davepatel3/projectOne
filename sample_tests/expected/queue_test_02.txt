[+][+]
4
[+][+][-][+]
93.5007
